### ISSUE TITLE
Test helper to simulate tabbing

### ISSUE THREAD

**Issue by mydea** [emberjs/ember-test-helpers#738]
I don't even know if that would really be doable, but it would be awesome to have a test helper to simulate tabbing through a document.

I recently wrote some code for focus trapping, and couldn't really find a way to test if everything works as expected. I tried to do e.g. `triggerKeyEvent(myElement, 'keydown', 'Tab')` but that did not actually tab to the next element. Calling `focus()` does focus it, but is not a real substitute for this type of test.

Having something like `tab(myElement)` and `tab(myElement, { backwards: true })` (or similar) would be very helpful for some a11y-related tests!

**Comment by dbollinger** [comment]
Finding myself looking for a similar solution today, specifically testing that a focus trap has been properly applied to my component.

**Comment by mydea** [comment]
Yeah, I did something similar to this in the end, which is OK I guess. But things like testing if the default behavior still works (e.g. if I tab on a non-first/last item, it moves to the next item), is not possible like this, I guess!

**Comment by rwjblue** [comment]
I'm _generally_ ok with something in this space, but I don't know what the best way to implement it would be...

**Comment by mydea** [comment]
Yeah, I sadly also don't really have an idea there, otherwise I'd have tried to do a PR myself - but maybe somebody out there _has_ an idea ;)
